src/test/java/com/slack/kaldb/logstore/search/KaldbDistributedQueryServiceTest.java

```diff
@@ -237,10 +237,10 @@ public void testSearch() {
                 .setBucketCount(2)
                 .build());
 
-    assertThat(searchResponse.getHitsCount()).isEqualTo(100);
-    assertThat(searchResponse.getTotalCount()).isEqualTo(200);
     assertThat(searchResponse.getTotalNodes()).isEqualTo(2);
     assertThat(searchResponse.getFailedNodes()).isEqualTo(0);
+    assertThat(searchResponse.getTotalCount()).isEqualTo(200);
```

Comment:
changing the order is to understand why KaldbDistributedQueryServiceTest fails ( which is not addressed in this PR as we're still trying to debug it )